integration-cli/docker_utils.go

```diff
@@ -86,6 +86,15 @@ func init() {
 	dockerBasePath = info.DockerRootDir
 	volumesConfigPath = filepath.Join(dockerBasePath, "volumes")
 	containerStoragePath = filepath.Join(dockerBasePath, "containers")
+	// Make sure in context of daemon, not the local platform. Note we can't
```

Comment:
given that we only add` "/volumes"` or `"\volumes"` and no other elements, would it be cleaner to (sorry Jess), just do a literal `+ separator + "volumes"` instead of replacing? Not very clean either, but, idk